### ISSUE TITLE
ID_INTAKE_XARELTO DEV and ID_INTAKE_INVOKANA

### ISSUE THREAD

**Issue by jesusitd** [intakedesk/PowerBI-General#320]
Prepared SQL statements for:
- Cases entity of Clients dataflow
- Leads entity of TV 2.0 dataflow
- Cases query for TV 2.0 dataset

This is not needed for Providers (no Xarelto critical data as far as seen in examination).

Just waiting on @fedegarza to let us know the migration date. Moved to Approval/Revision during this period.

CC: @CesarITD
